jib-core/src/test/java/com/google/cloud/tools/jib/cache/LayerEntriesSelectorTest.java

```diff
@@ -63,6 +67,29 @@
         .collect(ImmutableList.toImmutableList());
   }
 
+  @Test
+  public void testLayerEntryTemplate_fileModified() throws IOException {
+    TemporaryFolder temporaryFolder = new TemporaryFolder();
+    temporaryFolder.create();
+    Path layerFile = temporaryFolder.newFolder("testFolder").toPath().resolve("file");
+    LayerEntry layerEntry = new LayerEntry(layerFile, AbsoluteUnixPath.get("/extraction/path"));
+
+    LayerEntryTemplate layerEntryTemplate = new LayerEntryTemplate(layerEntry);
+    DescriptorDigest expectedSelector =
```

Comment:
This seems to be testing `LayerEntryTemplate` and duplicating the logic in `LayerEntriesSelector`. Should probably test `LayerEntriesSelector#generateSelector` instead.